package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "leaderboard-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "webpack --config webpack.config.js"
```

Comment:
your package file doesn't have some of the required scripts like build for this project
`"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
`